{{cookiecutter.project_slug}}/clients/web/vue3/src/views/ResetPassword.vue

```diff
@@ -29,11 +43,27 @@ export default {
   },
   setup() {
     const form = ref(new PasswordResetForm())
+    let matchingPasswordsError = ref(false)
+    const store = useStore()
     const route = useRoute()
+    const router = useRouter()
+
+    watch(
+      () => [form.value.password.value, form.value.passwordConfirmation.value],
+      () => {
+        if (form.value.password.value && form.value.passwordConfirmation.value
+            && form.value.password.value !== form.value.passwordConfirmation.value){
+          matchingPasswordsError.value = 'Passwords must match'
+        }else{
+          matchingPasswordsError.value = false
+        }
+      }
+    )
```

Comment:
@oudeismetis since you added this I can leave it here or ask Tomas to give it a shot!
We have a pre-build Validator for this [here](https://github.com/thinknimble/tn-forms/blob/2698ab22409678984034b533c51a879af2e11c78/src/validators.ts#L103) you can see an example of it in the `tn-forms` readme.md in the JS section [here]("https://github.com/thinknimble/tn-forms"). 

Check out the comment in the `forms.js`